selftest/cgroup/main.go

```diff
@@ -0,0 +1,96 @@
+package main
+
+import "C"
+
+import (
+	"encoding/binary"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"regexp"
+
+	bpf "github.com/aquasecurity/libbpfgo"
+)
+
+var reCgroup2Mount = regexp.MustCompile(`(?m)^cgroup2\s+(\S+)\s+`)
```

Comment:
Perhaps `(?m)^cgroup2\s\/[\S]+\scgroup2\s`, catching mount point start `/` and the type as `cgroup2`. WDYT?

```suggestion
var reCgroup2Mount = regexp.MustCompile(`(?m)^cgroup2\s\/[\S]+\scgroup2\s`)
```

Reply:
Or, going with your `FindStringSubmatch` approach, just set the regex to `(?m)^cgroup2\s\/(\S+)\scgroup2\s`. Be my guest.

Reply:
Yes. Your way is better.

Reply:
updated